packages/elements/src/Header/Header.tsx

```diff
@@ -48,6 +48,10 @@ export function Header(props: Props) {
 
   const isParentHeaderShown = React.useContext(HeaderShownContext);
 
+  // On models with Dynamic Island the status bar height is smaller than the safe area top inset.
+  const hasDynamicIsland = Platform.OS === 'ios' && insets.top > 50;
+  const statusBarHeight = hasDynamicIsland ? insets.top - 5 : insets.top;
+
```

Comment:
Is this value of 5 accurate? Because it **looks** 1 pixel short 
<img width="1376" alt="Screenshot 2023-10-24 at 8 56 40 AM" src="https://github.com/react-navigation/react-navigation/assets/80689446/2d724ba2-cf8f-4874-9c0f-0edb1aa19f3e">

https://github.com/react-navigation/react-navigation/issues/11655

Reply:
yup, it should probably be 5.33 instead of 5